### PR TITLE
AT_12.05_004 | Add link on GitHub

### PR DESCRIPTION
**Pull request by SvetlanaKanel**
https://trello.com/c/5cZozXga/978-%D0%B0%D1%821205004-freestyle-project-configurate-project-add-link-to-github

### CODE REVIEW

**Review by engineeryulia — APPROVED** [review]
LGTM

**Review by AlexMokeev — APPROVED** [review]

File: cypress/e2e/freestyleProjectConfigurate.cy.js [hunk 1]
```diff
@@ -18,4 +20,21 @@ describe('FreestyleProjectConfigurateProject', () => {
         cy.get('button[name=Submit]').click();
         cy.get('#description div:nth-child(1)').should('have.text', description);
     });
+
+    it('AT_12.05_004 | Add link on GitHub', () => {
+        cy.get('.jenkins-breadcrumbs__list-item [href="/"]').click();
+        cy.get('[href="job/MyProject1/"]').realHover();
+        cy.get('[href="job/MyProject1/"] .jenkins-menu-dropdown-chevron').click({force : true});
```
Comment (Rom-T) [inline cypress/e2e/freestyleProjectConfigurate.cy.js:27]: Try to find the locator without the project name

File: cypress/e2e/freestyleProjectConfigurate.cy.js [hunk 2]
```diff
@@ -18,4 +20,21 @@ describe('FreestyleProjectConfigurateProject', () => {
...
+        cy.get('[href="/job/MyProject1/configure"]').click();
+        cy.get('.jenkins-checkbox [name="githubProject"]').check({force : true});
+        cy.get('input[name="_.projectUrlStr"]').type(configurePageData.gitHubProjectURL);
+        cy.get('button[name="Submit"]').click();
+
+        cy.get('[href="https://github.com/RedRoverSchool/JenkinsQA_JS_06/"]')
+           .should('be.visible')
+           .click();
+        cy.url().should('include', 'RedRoverSchool/JenkinsQA_JS_06/'); 
+        cy.get('.author').should('include.text', 'RedRoverSchool');
```
Comment (Rom-T) [inline cypress/e2e/freestyleProjectConfigurate.cy.js:38]: Please move `data` in the fixtures

File: cypress/e2e/freestyleProjectConfigurate.cy.js (hunk 1 above)
Comment (SvetlanaKanel) [reply cypress/e2e/freestyleProjectConfigurate.cy.js]: done

File: cypress/e2e/freestyleProjectConfigurate.cy.js (hunk 2 above)
Comment (SvetlanaKanel) [reply cypress/e2e/freestyleProjectConfigurate.cy.js]: done

**Review by Rom-T — APPROVED** [review]

**Pull request merged**